networking_cisco/plugins/cisco/service_plugins/cisco_router_gbp_plugin.py

```diff
@@ -0,0 +1,128 @@
+# Copyright 2016 Cisco Systems, Inc.  All rights reserved.
```

Comment:
Yeah -- I've been avoiding changes to the core code for now so as not to affect the ASR team, and focusing on additions for our stuff. My hope is that we can make these optimizations going forward.